lib/cocoapods/generator/copy_resources_script.rb

```diff
@@ -154,20 +154,20 @@ def script
       ;;
   esac
 }
-EOS
-
-      RSYNC_CALL = <<EOS
-
-mkdir -p "${CONFIGURATION_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}"
-rsync -avr --copy-links --no-relative --exclude '*/.svn/*' --files-from="$RESOURCES_TO_COPY" / "${CONFIGURATION_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}"
-if [[ "${ACTION}" == "install" ]] && [[ "${SKIP_INSTALL}" == "NO" ]]; then
-  mkdir -p "${INSTALL_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}"
-  rsync -avr --copy-links --no-relative --exclude '*/.svn/*' --files-from="$RESOURCES_TO_COPY" / "${INSTALL_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}"
+SH
+
+      RSYNC_CALL = <<-SH
+if [ "$(cat "$RESOURCES_TO_COPY")" ]; then
```

Comment:
but `sh` isn't guaranteed to be bash, merely a POSIX-compliant shell. Since right now we use `/bin/sh`, I didn't feel it was worth changing it